annotation/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'java-library'
+apply plugin: 'kotlin'
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$rootProject.kotlinVersion"
```

Comment:
I have no clue about this, but why import JDK7 and not JDK8? I often get confused about java-versions, so just trying to understand

Reply:
This lib depends on the java version the project is targeting, which actually is `sourceCompatibility JavaVersion.VERSION_1_8` but since it was already using `kotlin-stdlib-jdk7` lib, I didn't change it.